addons/portal/static/src/js/skip_to_content.js

```diff
@@ -0,0 +1,58 @@
+/** @odoo-module **/
+
+import publicWidget from "@web/legacy/js/public/public_widget";
+
+publicWidget.registry.SkipToContent = publicWidget.Widget.extend({
+    selector: ".o_skip_to_content",
+
+    /**
+     * @override
+     */
+    start() {
+        this._observeBodyClass();
+        this._toggleSkipToContent();
+        return this._super(...arguments);
+    },
+    /**
+     * @override
+     */
+    destroy() {
+        this.observer?.disconnect();
+        this._super(...arguments);
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     */
+    _toggleSkipToContent() {
+        // The button should not be visible when there is no header present, as
+        // there would be nothing to skip. It should also be hidden when a modal is
+        // open.
+        const isModalPresent = document.querySelector("body")?.classList.contains("modal-open");
```

Comment:
Why do we need that? If the skip link is open and a modal is open, the skip link will just appear under the backdrop. What's the big deal about that?